libraries/ui-library/src/components/six-input/six-input.tsx

```diff
@@ -228,6 +228,18 @@ export class SixInput {
     return this.nativeInput?.setSelectionRange(selectionStart, selectionEnd, selectionDirection);
   }
 
+  /** Returns the start and end positions of the text selection */
+  @Method()
+  async getSelectionRange(): Promise<{
+    selectionStart: number | undefined | null;
+    selectionEnd: number | undefined | null;
+  }> {
+    return {
+      selectionStart: this.nativeInput?.selectionStart,
+      selectionEnd: this.nativeInput?.selectionEnd,
```

Comment:
I suggest that we also return the `selectionDirection`, as it can be set by the `setSelectionRange` method.

Reply:
Done